gradle-env.nix

```diff
@@ -326,15 +326,7 @@ let
 
   mkGradle = gradleSpec:
     gradleGen.gradleGen {
```

Comment:
Shouldn't this just be `gradleGen` or as of [01f4c7e](https://github.com/NixOS/nixpkgs/commit/01f4c7ec881a0a52638f090aacf51461a20b472f#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22R14604)?

Reply:
Actually `gradleGen` returns another function now so I think it should be `pkgs.callPackage (gradleGen { /* ... */ }) {};`

Reply:
Yeah. Looks like someone broke this again. It used to work.

Reply:
https://github.com/NixOS/nixpkgs/pull/119444

Reply:
Fixed.